Tokenomics Explainers - Minor Fixes (#206)
rocket-pool/RPIPs: tokenomics-explainers/002-rework-intro.md

Megapools will enable multiple validators to share the same Ethereum withdrawal address – this **reduces initial and ongoing gas costs** for node operators.

> **RPL issuance will be reduced from 5% to 1.5%**, because the current 3.5% inflation used for RPL rewards to NOs is no longer necessary.

Reviewer: ```suggestion
**RPL issuance will be reduced from 5% to 1.5%**, because the current 3.5% issuance used for RPL rewards to NOs is no longer necessary.
```

Author: Woops. Thanks @orangesamus

Forced delegate upgrades (with a grace period) will limit the technical debt the Rocket Pool protocol accumulates from previous versions.

## Personas

|Persona|Now|With Rework|
|-|-|-|

> |**Existing Rocket Pool Node Operator**|I need RPL to create minipools. I have to remain collateralized at 10% of RPL/ETH to receive RPL rewards. My commission is 14% of borrowed ETH.|I do not need RPL to create minipools. I will receive node operator commission paid in ETH (3.5%). If I *do* stake RPL I will also receive voter share rewards (5.0%) paid in ETH (with no 10% minimum or cliff). I will either benefit from surplus share value accrual to RPL (5.5%) or receive an additional (5.5%) of voter share rewards. I will no longer receive RPL rewards.|

Reviewer: ```suggestion
|**Existing Rocket Pool Node Operator**|I need RPL to create minipool validators. I have to remain collateralized at an RPL staked value of at least 10% of my borrowed ETH to receive RPL issuance rewards. My commission is 14% of my borrowed ETH.|I do not need RPL to create megapool validators. My commission is 3.5% of my borrowed ETH. If I choose to stake RPL (which makes it vote-eligible) I will receive additional ETH rewards from voter share (5.0% of protocol revenue from all megapools) proportional to my vote-eligible RPL. I do not have a minimum staked RPL requirement (or "cliff") to maintain eligibility for voter share rewards, but my max vote eligibility is an RPL staked value of 150% of my bonded ETH. I will also benefit from either surplus share value accrual to RPL (5.5% of protocol revenue from all megapools) or a 5.5% increase to voter share. I will no longer receive RPL issuance
```

Author: Going to take this and potentially cut it down. Getting worried about length.

> |**Existing Rocket Pool Node Operator**|I need RPL to create minipool validators. I have to remain collateralized at an RPL staked value of at least 10% of my borrowed ETH to receive RPL issuance rewards. My commission is 14% of my borrowed ETH.|I do not need RPL to create megapool validators. My commission is 3.5% of my borrowed ETH. If I choose to stake RPL I will receive additional ETH rewards (5% of protocol revenue from all megapools) proportional to my share of staked RPL (up to a maximum). I do not need to maintain a minimum stake to receive these rewards. I will benefit from value accrual to RPL (5.5% of protocol revenue from all megapools) or a 5.5% increase to the rewards for staking RPL. I will no longer receive rewards in RPL.

Reviewer: ```suggestion
|**Existing Rocket Pool Node Operator**|I need RPL to create minipool validators. I have to remain collateralized at an RPL staked value of at least 10% of my borrowed ETH to receive RPL issuance rewards. My commission is 14% of my borrowed ETH.|I do not need RPL to create megapool validators. My commission is 3.5% of my borrowed ETH. If I choose to stake RPL I will receive additional ETH rewards (5% of borrowed ETH revenue from all megapools) proportional to my share of staked RPL. I do not need to maintain a minimum stake to receive these rewards. I will benefit from value accrual to RPL (5.5% of borrowed ETH revenue from all megapools) or from this revenue being paid to staked RPL. I will no longer receive rewards in RPL.
```

Reviewer: Sorry for more changes... maybe we simplify by removing "up to a maximum" since "proportional" is vague enough to cover that (or you could keep it in there)? I realized "protocol revenue" is really just the 14% fee (not total borrowed ETH revenue since 86% of it belongs to rETH holders), so 5% of "protocol revenue" would only be 5% of the 14%. Similarly, a 5.5% increase could come across as increasing your rewards by 5.5% which isn't much. Instead it's 5.5% of borrowed ETH revenue also being paid to staked RPL.

I think this one is the most challenging to condense

> |**rETH holder**|My staking rewards are a function of Ethereum issuance and the commission paid to NOs. Currently 86% of solo staking rewards.|My staking rewards are a function of Ethereum issuance and the commission paid to the Rocket Pool protocol and then split by the pDAO to several targets. Initially unchanged at 86% of solo staking rewards but may be modified by future pDAO votes.|

Reviewer: ```suggestion
|**rETH holder**|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Currently my rewards are ~86% of solo staking APY since I am charged of a fee of ~14%. The fee is considered revenue for Rocket Pool and is paid to NOs as commission for their services.|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Initially my rewards are unchanged at ~86% of solo staking APY since I will still be charged of a fee of ~14%. The fee is considered revenue for Rocket Pool and will be split by the pDAO to several targets. The relative sizes and total sum of the revenue splits may be modified by future pDAO votes.|
```

> |**rETH holder**|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Currently my rewards are ~86% of solo staking APY. The fee is paid to NOs as commission for their services.|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Initially my rewards are unchanged at ~86% of solo staking APY. The fee is considered revenue for Rocket Pool and will be split by the pDAO to several targets. The total fee may be modified by future pDAO votes.|

Reviewer: ```suggestion
|**rETH holder**|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Currently my rewards are ~86% of solo staking APY since I am charged a fee of ~14%.|My staking rewards are a function of solo staking APY and the fee I am charged by Rocket Pool. Currently my rewards are ~86% of solo staking APY since I am still charged a total fee of ~14%. The total fee may be modified by future pDAO votes.|
```

Reviewer: If we want to really zoom in on rETH holders, they don't care what we do with the fee...

> |**Passive RPL Holder**|Holding RPL is speculation on the liquid staking narrative and the utility of RPL as a bond.|In addition to the liquid staking narrative, holding RPL is speculation that the tokenomics changes will cause rETH supply to grow and generate surplus ETH returns, which **may** be used to accrue value to RPL (via buy+burn or buy+LP) or may solely go to NOs staking RPL.|

Reviewer: ```suggestion
|**Passive RPL Holder**|Holding RPL is speculation on the liquid staking narrative and the utility of RPL as a bond.|In addition to the liquid staking narrative, holding RPL is speculation that the tokenomics changes will cause rETH supply to grow and generate surplus ETH revenue, which **may** be used to accrue value to RPL (via buy+burn or buy+LP) or may solely go to NOs staking RPL (through voter share rewards).|
```

> |**ETH-only Node Operator**|I cannot operate a node with Rocket Pool|I can now operate a node with Rocket Pool without exposure to RPL. I receive a 3.5% node operator commission on borrowed ETH. The lower my bond, the higher my rewards relative to solo staking. |

Reviewer: ```suggestion
|**ETH-only Node Operator**|I cannot operate a node with Rocket Pool|I can now operate a node with Rocket Pool without exposure to RPL. I initially receive a 3.5% node operator commission on my borrowed ETH. The 3.5% commission may be modified by future pDAO votes.|
```

Reviewer: Removed "the lower my bond...", since it also applies to "existing Rocket Pool NO", and is not something the Node Operator really has much "control" over, and is technically already covered under "on my borrowed ETH" (since lower bonds allow you to borrow more for the same capital). Could consider adding that back in to both places though if it would help to include that.
